tar_from_installation.sh

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env bash
 set -ex
+OUTDIR=$PWD/dist
+VERSION=`./version.sh`
```

Comment:
This line actual does work so maybe better leave it the else block below?

Reply:
oops yes, this was a late "cleanup" and I wasn't paying attention.  I'll undo